DigitalLearningSolutions.Web/Views/MyAccount/_CentreSpecificEmailDetailsSection.cshtml

```diff
@@ -0,0 +1,25 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.MyAccount
+@model CentreSpecificEmailDetailsViewModel
+
+@{ var key = Model.CentreName ?? "Centre email"; }
+<div class="nhsuk-summary-list__row basic-summary-list__row">
+  <dt class="nhsuk-summary-list__key">
+    @key
```

Comment:
Given that 
* this file is only used in one place
* `Model.CentreName` for that usage is always null
* If `Model.CentreName` were _not_ null, this would be a change in behaviour of the UI which is not specified by the ticket, and also wouldn't make much sense

I think that this should just always be the string "Centre email"

Reply:
This is used in 2 places, in `_CentreSpecificDetailsCard` and in `_CentreEmailsCard`. For the former, I create a `CentreSpecificEmailDetailsViewModel` with `CentreName = null` and "Centre email" as label. For the later I display as label the centre name